packages/typespec-ts/package.json

```diff
@@ -64,14 +64,14 @@
   "license": "MIT",
   "devDependencies": {
     "@azure-rest/core-client": "^2.3.1",
-    "@typespec/http-specs": "0.1.0-alpha.17-dev.2",
+    "@typespec/http-specs": "0.1.0-alpha.17",
     "@typespec/spector": "0.1.0-alpha.11-dev.2",
```

Comment:
you can use this https://www.npmjs.com/package/@typespec/spector/v/0.1.0-alpha.11